### PR TITLE
Fetch footer from AEM site

### LINKED ISSUE #46: Fetch Footer from AEM and render it in Franklin pages
As seen in https://www.zeiss.de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/europaeischer-erfinderpreis-2018.html

image

### PR DESCRIPTION
**Pull request by tmaret**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #46

Test URLs:
- Before: https://main--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/
- After: https://issue-46-2--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/

### CODE REVIEW

**Comment by tmaret** [comment]
There are a few layout issues but more importantly the breadcrumb is missing because we are fetching the footer from a fixed path.

**Review by sdmcraft — COMMENTED** [review]
Thanks @tmaret. Looks good except that currently the footer is using the styles from `styles.css` which is a straight copy of the styles from the main Zeiss site. The limited subset of needed styles are being put in `simplified-syles.css`. So for the footer, you may want to move all the styles it needs to either `simplified-styles.css` (globally reusable styles) or `footer.css` (styles specific to footer only)

**Comment by tmaret** [comment]
Thanks @sdmcraft ! For the breadcrumb, should we store it in each page .docx ? I'd track a separate issue to cover that part.

File: scripts/lib-franklin.js [hunk 1]
```diff
@@ -145,18 +145,12 @@ export function decorateIcons(element = document, isSprite = false) {
...
-        const iconHTML = await resp.text();
-        if (iconHTML.match(/<style/i)) {
-          const img = document.createElement('img');
-          img.src = `data:image/svg+xml,${encodeURIComponent(iconHTML)}`;
-          span.appendChild(img);
-        } else {
-          span.innerHTML = iconHTML;
-        }
-      }
+      span.querySelectorAll('svg > use[*|href]').forEach((xlinkRef) => {
```
[the remark this answers is not in the document]
Comment (tmaret) [reply scripts/lib-franklin.js:148]: That makes sense but (a) the `decorateIcons` method in the project `lib-franklin.js` was already patched with borken my PR fixes and (b) the [boilerplate](https://github.com/adobe/helix-project-boilerplate/blob/main/scripts/lib-franklin.js#L136) changed significantly (caching was added).

How about we capture updating `lib-franklin.js` and overriding from the latest boilerplate project in a dedicated ticket ?

[the remark this answers is not in the document]
Comment (tmaret) [reply scripts/lib-franklin.js:148]: https://github.com/hlxsites/zeiss/issues/123

**Comment by sdmcraft** [comment]
> Thanks @sdmcraft ! For the breadcrumb, should we store it in each page .docx ? I'd track a separate issue to cover that part.

If breadcrumb is the same for each page, it could be part of the footer itself. wdyt?

**Comment by tmaret** [comment]
Ok, I'll generate the breadcrumb in the footer block. We'll load it from content when required.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**